### PR TITLE
Whitespace consistency

### PR DESCRIPTION
**Pull request by passy**
Sorry buddy, I can't help myself ...

### CODE REVIEW

File: undo-manager.html [hunk 1]
```diff
@@ -174,13 +176,13 @@
...
+       *
        * @method stackUI
        */
-      stackUI: function(){
-        !this.stack.canUndo()? this.$.undo.disabled = true : this.$.undo.disabled = false;
-        !this.stack.canRedo()? this.$.redo.disabled = true : this.$.redo.disabled = false;
-        this.stack.dirty()? this.$.dirty.style.display = 'none': 'block';
+      stackUI: function () {
+        !this.stack.canUndo() ? this.$.undo.disabled = true : this.$.undo.disabled = false;
+        !this.stack.canRedo() ? this.$.redo.disabled = true : this.$.redo.disabled = false;
```
Comment (sindresorhus) [inline undo-manager.html:184]: Unrelated, but uhm:

``` js
this.$.redo.disabled = !this.stack.canRedo();
```

File: undo-manager.html [hunk 2]
```diff
@@ -174,13 +176,13 @@
...
+        this.stack.dirty() ? this.$.dirty.style.display = 'none': 'block';
```
Comment (sindresorhus) [inline undo-manager.html:185]: this doesn't work

Comment (addyosmani) [inline undo-manager.html:185]: Thanks for flagging that. I'll get it fixed.

**Comment by addyosmani** [comment]
Thanks @passy!

**Pull request merged**